packages/design-system/src/components/Tab/style.scss

```diff
@@ -38,9 +38,10 @@
 
   &--selected {
     font-weight: 700;
+    color: $primary-color;
 
     .tab__description {
-      color: #8e949a;
+      color: $primary-color;
```

Comment:
Should't it be $secondary-color and only on hover $primary-color?

Reply:
this is description in selected state and I talked about it with Jo -> she decided to go for primary-color